app/cafe.py

```diff
@@ -0,0 +1,26 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError, NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError(f"{visitor['name']} is not"
+                                     f" vaccinated and cannot visit the cafe.")
+
+        expiration_date = visitor["vaccine"].get("expiration_date")
+        if expiration_date < datetime.date.today():
```

Comment:
The `expiration_date` retrieved from the visitor's vaccine information should be converted to a `datetime.date` object before comparison. Ensure that `expiration_date` is in the correct format for comparison with `datetime.date.today()`.